packages/expo-yarn-workspaces/common/cross-spawn-sync.js

```diff
@@ -0,0 +1,18 @@
+"use strict";
+
+const child_process = require("child_process");
+const os = require("os").platform();
+
+let selectiveCmdify = cmd => cmd;
+if (os === "win32") {
+  selectiveCmdify = cmd => cmd + ".cmd";
+}
+
+module.exports = function spawnSync(cmd, args, options) {
+  cmd = selectiveCmdify(cmd);
+  if (!Array.isArray(args)) {
```

Comment:
remove this, make it the caller's responsibility to pass in an array

Reply:
done

Reply:
One sec

Reply:
Now, it is done